docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md

```diff
@@ -192,7 +237,7 @@ await api.transact(
                         },
                         sale_shares: [],
                         maximum_uos_payment: '2.00000000 UOS',
-                        group_restriction: [],
+                        group_restriction: [2305843009213693953, 3458764513820540932],
```

Comment:
If this is the above case 5, `3458764513820540932` should be `3458764513820540930`

Reply:
fixed the calculation. it's my intention to show different parameter combinations, they don't have to be the same. anyway,I changed them to be the same